tests/testthat/test-salmon.R

```diff
@@ -0,0 +1,23 @@
+test_that('Test salmon and getSalmonMetadata', {
```

Comment:
If we're only comparing the file lists, I don't want to keep the entire salmon_output directory in the repo.

Reply:
Are you talking about the `data/salmon_output_exp` directory? That isn't used for this test, it's used for the `tximport` tests. See that script. I think it's necessary to keep.